### PR TITLE
fix!: Upgrade tflint from v0.47.0 to v0.48.0

### PR DESCRIPTION
**Pull request by sebastianczech**
## Description

PR upgrades version of `tflint`.

## Motivation and Context

While working on AWS modules, there were issues detected in newer version of `tflint` and described in https://github.com/PaloAltoNetworks/terraform-aws-vmseries-modules/pull/396. 

## How Has This Been Tested?

`tflint` in version `0.48.0` was tested locally while working on https://github.com/PaloAltoNetworks/terraform-aws-vmseries-modules/pull/396.

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by sebastianczech** [comment]
Please do not merge until merging https://github.com/PaloAltoNetworks/terraform-aws-vmseries-modules/pull/396

**Review by FoSix — APPROVED** [review]

**Pull request merged**
